uspv/init.go

```diff
@@ -117,9 +148,13 @@ func (s *SPVCon) Connect(remoteNode string) error {
 		return fmt.Errorf("Remote node version: %x too old, exiting.", mv.ProtocolVersion)
 	}
 
-	if strings.Contains(mv.UserAgent, "ABC") {
-		// if we connected through a DNS Peer and it doesn't implement service bit filtering
-		return fmt.Errorf("Remote node %s invalid", mv.UserAgent)
+	if !(strings.Contains(mv.UserAgent, "Satoshi") || strings.Contains(mv.UserAgent, "btcd")) && (len(seedAdrs) != 0) {
```

Comment:
whitelists are ugly and we should filter based on behavior, not based on names, but we can leave this for now until the connection management is more built-out.

Reply:
len(seedAdrs) isn't relevant here; the handshake works or doesn't regardless